### DISCUSSION TITLE
cdh 集成doris的时候，启动doris会报错org.apache.doris.PaloFe

### DISCUSSION THREAD

**Discussion by Tangsonghuai** [apache/doris#35481] [RFC]
我在使用cdh集成doris2的时候，fe启动的时候，报错
找不到或无法加载主类 org.apache.doris. "PaloFe"
我不知道怎么解决这个问题

启动脚本在这里

`    if [[ $DORIS_FE_FIRST_LIVE = "true" ]];
    then
         log "Frist Starting the Doris Frontend, MasterIP:$DORIS_FE_MASTER_IP ; port :$DORIS_FE_MASTER_PORT"
         exec $JAVA $final_java_opt -XX:OnOutOfMemoryError="kill -9 %p" org.apache.doris.PaloFe -helper $DORIS_FE_MASTER_IP:$DORIS_FE_MASTER_PORT
    else
         log "Starting the Doris Frontend"
         exec  $JAVA $final_java_opt -XX:OnOutOfMemoryError="kill -9 %p" org.apache.doris.PaloFe
    fi
    ;;`

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题
